app/services/katello/pulp3/repository_mirror.rb

```diff
@@ -166,21 +166,13 @@ def remote_feed_url
         uri.to_s
       end
 
-      def publication_options(repository_version)
-        popts = {repository_version: repository_version}
-        if (type_specific_options = repo_service.try(:mirror_publication_options))
-          popts.merge!(type_specific_options)
-        end
-        popts
-      end
-
       def create_publication
         if (href = version_href)
           if repo_service.repo.content_type == "deb"
             publication_data = api.publication_verbatim_class.new({repository_version: href})
             api.publications_verbatim_api.create(publication_data)
           else
-            publication_data = api.publication_class.new(publication_options(href))
+            publication_data = api.publication_class.new(repository_version: href)
```

Comment:
This essentially reverts to what we had before 5876a01660ff0c1696b7acf13969e625889931d4.